compiler/codegen/Relocation.cpp

```diff
@@ -481,6 +481,8 @@ const char *TR::ExternalRelocation::_externalRelocationTargetKindNames[TR_NumExt
    "TR_MethodCallAddress (99)",
    "TR_DiscontiguousSymbolFromManager (100)",
    "TR_ResolvedTrampolines (101)",
+   "TR_BlockFrequency (102)",
+   "TR_RecompQueuedFlag (103)",
```

Comment:
There is some conflict shaping up between this commit and the first commit in https://github.com/eclipse/omr/pull/5729/commits since they both touch the same code.